README.md

```diff
@@ -28,33 +28,33 @@ Certificate Generator aims to automatically generate certificates based on the d
 * _**For Candidates:**_
     i. The dashboard will contain thumbnails of all the certificates received by the student.
     ii. There will be an option to view a particular certificate.
-    iii. The candidate will also be allowed to share the certificate.
+    iii. The candidate will also be allowed to share the certificate with others on any social platform.
```

Comment:
we are yet to take this as a functionality, hence it was not a part of readme

Reply:
Again, it would be difficult for us as well as you to keep track of changes in the PR without proper description. I request you to add detailed description wherever possible. Thanks